core/primitives/src/runtime/snapshots/near_primitives__runtime__config_store__tests__0.json.snap

```diff
@@ -172,7 +172,7 @@ expression: config_view
     },
     "grow_mem_cost": 1,
     "regular_op_cost": 3856371,
-    "vm_kind": "Wasmer0",
+    "vm_kind": "vm_kind",
```

Comment:
You will need to run the relevant test repeatedly to update the snapshots for all the protocol versions.